tests/scripts/all.sh

```diff
@@ -185,6 +185,10 @@ pre_initialize_variables () {
     # Include more verbose output for failing tests run by CMake or make
```

Comment:
What about Windows? It's not uncommon to have warnings that are specific to MSVC. It wouldn't be a big deal to fix them later, but that defeats the purpose of having a full CI with the new bignum.